spec/helper/db.helper.js

```diff
@@ -1,12 +1,12 @@
 const fs = require("fs").promises;
 const path = require("path");
 
-const ParkingSlot = require("../../models/parking-slot");
+const Slot = require("../../models/slot");
 
 
 exports.newDB = async () => {
   try {
-    const lot = [...Array(10).keys()].map((i) => new ParkingSlot(i));
+    const lot = [...Array(10).keys()].map((i) => new Slot(i));
```

Comment:
The feedback item is pending

Reply:
I still see that we are using the number `10` here and in a lot of other places instead of the constant. Did you work ok moving the parking lot size to a constant? Reach out to me if you have questions regarding this review point.

> 

> Move the size of the Array to a constant which can be used in all places, instead of repetitive usage of the magic number 10 in every location. This will help reduce the number of changes required if we need to increase or decrease the parking lot size at any point in time.